charts/nexus-iq/templates/test/test-check-logs.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-test-check-logs"
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+spec:
+  containers:
+    - name: {{ .Release.Name }}-test-check-logs
+      image: busybox
+      command: ["/bin/sh"]
+      args: ["-c", "cat /var/log/nexus-iq-server/clm-server.log | grep {{ .Chart.AppVersion }} || exit 1"]
+      volumeMounts:
+        - mountPath: /var/log/nexus-iq-server
+          name: nexus-logs
+  volumes:
+    - name: nexus-logs
+      persistentVolumeClaim:
+        claimName: {{ .Values.persistence.existingLogClaim | default (printf "%s-%s" (include "iqserver.fullname" .) "log") }}
+  restartPolicy: Never
```

Comment:
this is cool. if i'm reading it correctly, this pod will mount the running server's logs and grep for the expected app version? nice.